repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -27,14 +27,17 @@ stage("Performance Test") {
     def performanceJenkinsJob = "performance-test"
     def performanceTestParam = "perf_test"
     def extraOptionsParam = "extra_options"
+    def random = new Random()
 
     def jobsToRun = [:]
 
     for (int index = 0; index < perfTests.size(); index++) {
         def perfTest = perfTests[index]
         jobsToRun[perfTest] = {
             retry(3) {
-                build(job: performanceJenkinsJob, parameters: [
+                build(job: performanceJenkinsJob,
+                      quietPeriod: (random.nextInt() % 585) + 15,
```

Comment:
That's true looking at the contract. it seems to just fire off immediately in that case, based on the behavior i saw earlier. do we actually need the 15 second minimum quiet period? I can trivially set it to anything within 10 minutes or i can keep this plus add an absolute value to it.